2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200756-320g69q5\logs\debug.log
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200756-320g69q5\logs\debug-internal.log
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000204C2AE0E10>
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:init():370] calling init triggers
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 350fpbwo
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/350fpbwo
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-01 20:07:56,740 INFO    MainThread:5352 [wandb_run.py:_restore():1566] restore
2021-07-01 20:07:57,699 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103622
  total_bytes: 103622
}

2021-07-01 20:07:57,854 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103622
  total_bytes: 103622
}

2021-07-01 20:07:58,162 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103622
  total_bytes: 103622
}

2021-07-01 20:07:58,459 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103622
  total_bytes: 107692
}

2021-07-01 20:07:58,787 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103622
  total_bytes: 107692
}

2021-07-01 20:07:58,897 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,007 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,116 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,225 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,335 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,444 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:07:59,553 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107692
  total_bytes: 107692
}

2021-07-01 20:08:00,670 INFO    MainThread:5352 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-01 20:08:00,670 INFO    MainThread:5352 [wandb_run.py:_show_history():1909] rendering history
2021-07-01 20:08:00,685 INFO    MainThread:5352 [wandb_run.py:_show_files():1938] logging synced files
2021-07-01 20:08:00,685 INFO    MainThread:5352 [wandb_init.py:init():419] starting backend
2021-07-01 20:08:00,685 INFO    MainThread:5352 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-01 20:08:00,701 INFO    MainThread:5352 [backend.py:ensure_launched():135] starting backend process...
2021-07-01 20:08:00,857 INFO    MainThread:5352 [backend.py:ensure_launched():140] started backend process with pid: 10668
2021-07-01 20:08:00,857 INFO    MainThread:5352 [wandb_init.py:init():424] backend started and connected
2021-07-01 20:08:00,888 INFO    MainThread:5352 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-01 20:08:00,888 INFO    MainThread:5352 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-01 20:08:00,888 INFO    MainThread:5352 [wandb_init.py:init():472] updated telemetry
2021-07-01 20:08:00,888 INFO    MainThread:5352 [wandb_init.py:init():491] communicating current version
2021-07-01 20:08:03,429 INFO    MainThread:5352 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-01 20:08:03,429 INFO    MainThread:5352 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-01 20:08:03,882 INFO    MainThread:5352 [wandb_init.py:init():529] starting run threads in backend
2021-07-01 20:08:04,334 INFO    MainThread:5352 [wandb_run.py:_console_start():1624] atexit reg
2021-07-01 20:08:04,334 INFO    MainThread:5352 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-01 20:08:04,334 INFO    MainThread:5352 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-01 20:08:04,334 INFO    MainThread:5352 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-01 20:08:04,334 INFO    MainThread:5352 [wandb_init.py:init():554] run started, returning control to user process
2021-07-01 20:08:06,100 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-01 20:08:35,072 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-01 20:08:40,252 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-01 20:09:57,029 INFO    MainThread:5352 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-01 20:09:57,029 INFO    MainThread:5352 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-01 20:42:13,045 INFO    MainThread:5352 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-01 20:42:51,297 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 20:42:51,297 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:42:51,297 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:42:51,298 INFO    MainThread:5352 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_204251-2c7i87bz\logs\debug.log
2021-07-01 20:42:51,298 INFO    MainThread:5352 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_204251-2c7i87bz\logs\debug-internal.log
2021-07-01 20:42:51,299 INFO    MainThread:5352 [wandb_init.py:init():370] calling init triggers
2021-07-01 20:42:51,299 INFO    MainThread:5352 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 20:42:51,299 INFO    MainThread:5352 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 320g69q5
2021-07-01 20:42:51,301 INFO    MainThread:5352 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/320g69q5
